combat-trainer.lic

```diff
@@ -138,7 +138,19 @@ class SetupProcess
       echo('skipping summoned weapons because no moonblade available') if $debug_mode_ct
       weapon_training = weapon_training.reject { |skill, _| game_state.summoned_info(skill) }
     end
-    new_weapon_skill = weapon_training.min_by { |skill, _| [DRSkill.getxp(skill), DRSkill.getrank(skill)] }.first
+
+    new_weapon_skill=""
+    echo "Priority weapons: #{get_settings.priority_weapons}" if $debug_mode_ct
+    get_settings.priority_weapons.each do | weapon |
+      if DRSkill.getxp(weapon)==0 && weapon_training[weapon]
+        new_weapon_skill=weapon
```

Comment:
Should it `break` after finding a priority weapon? I think this would make the _last_ item in the last the _highest_ priority, which seems backwards.